src/phoenix/server/api/input_types/InvocationParameters.py

```diff
@@ -15,6 +15,7 @@ class CanonicalParameterName(str, Enum):
     RANDOM_SEED = "random_seed"
     TOOL_CHOICE = "tool_choice"
     RESPONSE_FORMAT = "response_format"
+    REASONING_EFFORT = "reasoning_effort"
```

Comment:
do we need this to be canonical for the time being? I don't think there's any other model that shares a similar concept?

Reply:
Does it hurt anything?

Reply:
nope!